commit 1ec18f3a8d40d2b3ec331f62a2897c5508f2f581 (refs/changes/58/5458/1)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-01-31T16:54:04-05:00 (8 months ago)
    
    TRITON-1092 sdc-system-tests "sdc-system-tests.tap" not in TAP format

diff --git a/runtests b/runtests
index 5c9b1f4..7bf2e97 100755
--- a/runtests
+++ b/runtests
@@ -66,7 +66,7 @@ NODEUNIT=./node_modules/nodeunit/bin/nodeunit
 
 # Options.
 opt_test_pattern=
-opt_reporter=default
+opt_reporter=tap
 while getopts "hf:r:" opt
 do
     case "$opt" in
